### PR TITLE
fix(chat): fix Markdown-inline images encoded as Data URLs (Issue #1518)

### PR DESCRIPTION
**Pull request by IlyaBondar**
**Description:**

 fix Markdown-inline images encoded as Data URLs

Issues:

- Issue #1518 

**UI changes**

image

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1519.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/utils/app/attachments.ts [hunk 1]
```diff
@@ -4,7 +4,11 @@ export const getMappedAttachmentUrl = (url: string | undefined) => {
   if (!url) {
     return undefined;
   }
-  return url.startsWith('//') || url.startsWith('http') ? url : `api/${url}`;
+  return url.startsWith('data:') ||
```
Comment (Alexander-Kezik) [inline apps/chat/src/utils/app/attachments.ts]: what about `file:` `ftp:` etc.?

File: apps/chat/src/utils/app/attachments.ts [hunk 2]
```diff
@@ -4,7 +4,11 @@ export const getMappedAttachmentUrl = (url: string | undefined) => {
...
+    url.startsWith('//') ||
+    url.startsWith('http')
```
Comment (Alexander-Kezik) [inline apps/chat/src/utils/app/attachments.ts]: maybe it's better to check if url starts with `http[s]:` with regex?

**Review by Alexander-Kezik — DISMISSED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1519.nightly-test.deltixhub.io
>E2E tests status: success

**Review by Alexander-Kezik — APPROVED** [review]

**Pull request merged**